packages/icons/src/library/post-date.js

```diff
@@ -5,16 +5,8 @@ import { Path, SVG } from '@wordpress/primitives';
 
 const postDate = (
 	<SVG fill="none" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
```

Comment:
There's one fill left here. Removing it just makes this a tiny bit smaller. No other effect.